### DISCUSSION TITLE
Build/Plugins/GenericEditor

### DISCUSSION THREAD

**Discussion by dikshauat** [Sunbird-Ed/Community#199] [General Discussion]
Hi @keshavprasadms, @rahuls0ni,

As mentioned in the previous post (https://github.com/Sunbird-Ed/Community/discussions/97) tried Generic Editor with release 4.10.1_RC1 but not getting success. Below is the console output of the job. Kindly check it and please help me resolving the issue

Started by user [Diksha POC](http://35.223.161.5:8080/user/dikshapoc)
Checking out git https://github.com/project-sunbird/sunbird-generic-editor.git into /var/lib/jenkins/workspace/Build/Plugins/GenericEditor@script/a4bd6e68954a25239b55c35ed8919f45766482f2bbd688447c591a1bbf5b1e3e to read Jenkinsfile
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/Build/Plugins/GenericEditor@script/a4bd6e68954a25239b55c35ed8919f45766482f2bbd688447c591a1bbf5b1e3e/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-generic-editor.git # timeout=10
Fetching upstream changes from https://github.com/project-sunbird/sunbird-generic-editor.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress -- https://github.com/project-sunbird/sunbird-generic-editor.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse origin/release-4.10.1^{commit} # timeout=10
Checking out Revision 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae (origin/release-4.10.1)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae # timeout=10
Commit message: "Issue #KN-133 merge: Merge pull request #72 from vaibhavbhuva/release-4.10.1"
 > git rev-list --no-walk 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae # timeout=10
[Pipeline] Start of Pipeline
[Pipeline] node
Running on [Jenkins](http://35.223.161.5:8080/computer/(built-in)/) in /var/lib/jenkins/workspace/Build/Plugins/GenericEditor
[Pipeline] {
[Pipeline] ansiColor
[Pipeline] {

[Pipeline] stage
[Pipeline] { (Checkout)
[Pipeline] cleanWs
[WS-CLEANUP] Deleting project workspace...
[WS-CLEANUP] Deferred wipeout is used...
[WS-CLEANUP] done
[Pipeline] checkout
The recommended git tool is: NONE
No credentials specified
Cloning the remote Git repository
Cloning repository https://github.com/project-sunbird/sunbird-generic-editor.git
 > git init /var/lib/jenkins/workspace/Build/Plugins/GenericEditor # timeout=10
Fetching upstream changes from https://github.com/project-sunbird/sunbird-generic-editor.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress -- https://github.com/project-sunbird/sunbird-generic-editor.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-generic-editor.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
Avoid second fetch
 > git rev-parse origin/release-4.10.1^{commit} # timeout=10
Checking out Revision 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae (origin/release-4.10.1)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae # timeout=10
Commit message: "Issue #KN-133 merge: Merge pull request #72 from vaibhavbhuva/release-4.10.1"
[Pipeline] checkout
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-generic-editor.git # timeout=10
Fetching upstream changes from https://github.com/project-sunbird/sunbird-generic-editor.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress -- https://github.com/project-sunbird/sunbird-generic-editor.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse refs/remotes/origin/refs/tags/release-4.10.1_RC1^{commit} # timeout=10
 > git rev-parse refs/tags/release-4.10.1_RC1^{commit} # timeout=10
Checking out Revision 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae (refs/tags/release-4.10.1_RC1)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae # timeout=10
Commit message: "Issue #KN-133 merge: Merge pull request #72 from vaibhavbhuva/release-4.10.1"
[Pipeline] sh
+ git rev-parse --short HEAD
[Pipeline] echo
github_release_tag specified, building from github_release_tag: release-4.10.1_RC1
[Pipeline] sh
+ git clone https://github.com/project-sunbird/sunbird-content-plugins.git plugins
Cloning into 'plugins'...
[Pipeline] sh
+ cd plugins
+ git ls-remote --tags origin release-4.10*
+ sort -V
+ grep -o release-4.10.*
+ tail -n1
+ checkout_tag=
+ git checkout tags/ -b
error: switch `b' requires a value
usage: git checkout [<options>] <branch>
   or: git checkout [<options>] [<branch>] -- <file>...

    -q, --quiet           suppress progress reporting
    -b <branch>           create and checkout a new branch
    -B <branch>           create/reset and checkout a branch
    -l                    create reflog for new branch
    --detach              detach HEAD at named commit
    -t, --track           set upstream info for new branch
    --orphan <new-branch>
                          new unparented branch
    -2, --ours            checkout our version for unmerged files
    -3, --theirs          checkout their version for unmerged files
    -f, --force           force checkout (throw away local modifications)
    -m, --merge           perform a 3-way merge with the new branch
    --overwrite-ignore    update ignored files (default)
    --conflict <style>    conflict style (merge or diff3)
    -p, --patch           select hunks interactively
    --ignore-skip-worktree-bits
                          do not limit pathspecs to sparse entries only
    --ignore-other-worktrees
                          do not check if another worktree is holding the given ref
    --recurse-submodules[=<checkout>]
                          control recursive updating of submodules
    --progress            force progress reporting

[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
ERROR: script returned exit code 129
Finished: FAILURE

**Comment by beepdot** [comment]
@vinukumar-vs @vaibhavbhuva Can you please assist here?

**Reply by dikshauat** [reply]
@keshavprasadms @vinukumar-vs @vaibhavbhuva , 
We tried executing the job with releas-5.0.0 . Getting the below error on Jenkins. Kindly assist us 


Started by user [Diksha POC](http://35.223.161.5:8080/user/dikshapoc)
Checking out git https://github.com/project-sunbird/sunbird-generic-editor.git into /var/lib/jenkins/workspace/Build/Plugins/GenericEditor@script/a4bd6e68954a25239b55c35ed8919f45766482f2bbd688447c591a1bbf5b1e3e to read Jenkinsfile
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/Build/Plugins/GenericEditor@script/a4bd6e68954a25239b55c35ed8919f45766482f2bbd688447c591a1bbf5b1e3e/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-generic-editor.git # timeout=10
Fetching upstream changes from https://github.com/project-sunbird/sunbird-generic-editor.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress -- https://github.com/project-sunbird/sunbird-generic-editor.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse origin/release-5.0.0^{commit} # timeout=10
Checking out Revision da532a1436c4f9fd79c948e6cb32abc53269e794 (origin/release-5.0.0)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f da532a1436c4f9fd79c948e6cb32abc53269e794 # timeout=10
Commit message: "Issue #SB-30910 merge: Merge pull request #76 from vaibhavbhuva/release-5.0.0"
 > git rev-list --no-walk da532a1436c4f9fd79c948e6cb32abc53269e794 # timeout=10
[Pipeline] Start of Pipeline
[Pipeline] node
Running on [Jenkins](http://35.223.161.5:8080/computer/(built-in)/) in /var/lib/jenkins/workspace/Build/Plugins/GenericEditor
[Pipeline] {
[Pipeline] ansiColor
[Pipeline] {

[Pipeline] stage
[Pipeline] { (Checkout)
[Pipeline] cleanWs
[WS-CLEANUP] Deleting project workspace...
[WS-CLEANUP] Deferred wipeout is used...
[WS-CLEANUP] done
[Pipeline] checkout
The recommended git tool is: NONE
No credentials specified
Cloning the remote Git repository
Cloning repository https://github.com/project-sunbird/sunbird-generic-editor.git
 > git init /var/lib/jenkins/workspace/Build/Plugins/GenericEditor # timeout=10
Fetching upstream changes from https://github.com/project-sunbird/sunbird-generic-editor.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress -- https://github.com/project-sunbird/sunbird-generic-editor.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-generic-editor.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
Avoid second fetch
 > git rev-parse origin/release-5.0.0^{commit} # timeout=10
Checking out Revision da532a1436c4f9fd79c948e6cb32abc53269e794 (origin/release-5.0.0)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f da532a1436c4f9fd79c948e6cb32abc53269e794 # timeout=10
Commit message: "Issue #SB-30910 merge: Merge pull request #76 from vaibhavbhuva/release-5.0.0"
[Pipeline] checkout
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-generic-editor.git # timeout=10
Fetching upstream changes from https://github.com/project-sunbird/sunbird-generic-editor.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress -- https://github.com/project-sunbird/sunbird-generic-editor.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse origin/release-5.0.0^{commit} # timeout=10
Checking out Revision da532a1436c4f9fd79c948e6cb32abc53269e794 (origin/release-5.0.0)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f da532a1436c4f9fd79c948e6cb32abc53269e794 # timeout=10
Commit message: "Issue #SB-30910 merge: Merge pull request #76 from vaibhavbhuva/release-5.0.0"
[Pipeline] sh
+ git rev-parse --short HEAD
[Pipeline] echo
github_release_tag specified, building from github_release_tag: release-5.0.0
[Pipeline] sh
+ git clone https://github.com/project-sunbird/sunbird-content-plugins.git plugins
Cloning into 'plugins'...
[Pipeline] sh
+ cd plugins
+ git ls-remote --tags origin release-5.0*
+ grep -o release-5.0.*
+ sort -V
+ tail -n1
+ checkout_tag=release-5.0.0_RC3
+ git checkout tags/release-5.0.0_RC3 -b release-5.0.0_RC3
Switched to a new branch 'release-5.0.0_RC3'
[Pipeline] echo
artifact_version: release-5.0.0_da532a1
[Pipeline] stage
[Pipeline] { (Build)
[Pipeline] sh
+ chmod 777 build.sh
[Pipeline] sh
+ bash ./build.sh release-5.0 da532a1 false
runTest:  false
Starting editor build from build.sh
v10.24.1 is already installed.
Now using node v10.24.1 (npm v6.14.12)
Now using node v10.24.1 (npm v6.14.12)
Reading package lists...
Building dependency tree...
Reading state information...
build-essential is already the newest version (12.4ubuntu1).
libpng-dev is already the newest version (1.6.34-1ubuntu0.18.04.2).
0 upgraded, 0 newly installed, 0 to remove and 4 not upgraded.
v10.24.1
6.14.12

> iltorb@2.4.5 install /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/iltorb
> node ./scripts/install.js || node-gyp rebuild

info looking for cached prebuild @ /var/lib/jenkins/.npm/_prebuilds/2a34dd-iltorb-v2.4.5-node-v64-linux-x64.tar.gz
info found cached prebuild 
info unpacking @ /var/lib/jenkins/.npm/_prebuilds/2a34dd-iltorb-v2.4.5-node-v64-linux-x64.tar.gz
info unpack resolved to /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/iltorb/build/bindings/iltorb.node
info unpack required /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/iltorb/build/bindings/iltorb.node successfully
info install Successfully installed iltorb binary!

> node-sass@4.14.1 install /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/node-sass
> node scripts/install.js

Cached binary found at /var/lib/jenkins/.npm/node-sass/4.14.1/linux-x64-64_binding.node

> phantomjs-prebuilt@2.1.16 install /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/karma-phantomjs-launcher/node_modules/phantomjs-prebuilt
> node install.js

PhantomJS not found on PATH
Download already available at /tmp/phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2
Verified checksum of previously downloaded file
Extracting tar contents (via spawned process)
Removing /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/karma-phantomjs-launcher/node_modules/phantomjs-prebuilt/lib/phantom
Copying extracted folder /tmp/phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2-extract-1664446459167/phantomjs-2.1.1-linux-x86_64 -> /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/karma-phantomjs-launcher/node_modules/phantomjs-prebuilt/lib/phantom
Writing location.js file
Done. Phantomjs binary available at /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/karma-phantomjs-launcher/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs

> phantomjs@2.1.7 install /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/phantomjs
> node install.js

PhantomJS not found on PATH
Download already available at /tmp/phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2
Verified checksum of previously downloaded file
Extracting tar contents (via spawned process)
Removing /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/phantomjs/lib/phantom
Copying extracted folder /tmp/phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2-extract-1664446462872/phantomjs-2.1.1-linux-x86_64 -> /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/phantomjs/lib/phantom
Writing location.js file
Done. Phantomjs binary available at /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/phantomjs/lib/phantom/bin/phantomjs

> core-js@2.6.12 postinstall /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> node-sass@4.14.1 postinstall /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/node-sass
> node scripts/build.js

Binary found at /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/node-sass/vendor/linux-x64-64/binding.node
Testing binary
Binary is fine

> cwebp-bin@4.0.0 postinstall /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/cwebp-bin
> node lib/install.js

  ✔ cwebp pre-build test passed successfully

> gifsicle@3.0.4 postinstall /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/gifsicle
> node lib/install.js

  ✔ gifsicle pre-build test passed successfully

> jpegtran-bin@3.2.0 postinstall /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/jpegtran-bin
> node lib/install.js

  ✔ jpegtran pre-build test passed successfully

> mozjpeg@5.0.0 postinstall /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/mozjpeg
> node lib/install.js

  ✔ mozjpeg pre-build test passed successfully

> optipng-bin@3.1.4 postinstall /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/optipng-bin
> node lib/install.js

  ✔ optipng pre-build test passed successfully

> pngquant-bin@4.0.0 postinstall /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/pngquant-bin
> node lib/install.js

  ⚠ The `/var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/pngquant-bin/vendor/pngquant` binary doesn't seem to work correctly
  ⚠ pngquant pre-build test failed
  ℹ compiling from source
  ✔ pngquant pre-build test passed successfully
  ✔ pngquant built successfully

> purifycss-webpack@0.7.0 postinstall /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/purifycss-webpack
> node lib/post_install.js

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/watchpack/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/postcss-cli/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 2439 packages from 1458 contributors and audited 2453 packages in 39.598s

84 packages are looking for funding
  run `npm fund` for details

found 271 vulnerabilities (44 low, 74 moderate, 127 high, 26 critical)
  run `npm audit fix` to fix them, or `npm audit` for details
./build.sh: line 23: bower: command not found
[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
ERROR: script returned exit code 127
Finished: FAILURE

**Reply by HarishGangula** [reply]
@dikshauat  Please check https://github.com/orgs/Sunbird-Ed/discussions/88#discussioncomment-3453311

**Reply by dikshauat** [reply]
@HarishGangula , this is what we got on using release-4.10.1_RC1 as suggested in https://github.com/orgs/Sunbird-Ed/discussions/88#discussioncomment-3453311

Started by user [Diksha POC](http://35.223.161.5:8080/user/dikshapoc)
Checking out git https://github.com/project-sunbird/sunbird-generic-editor.git into /var/lib/jenkins/workspace/Build/Plugins/GenericEditor@script/a4bd6e68954a25239b55c35ed8919f45766482f2bbd688447c591a1bbf5b1e3e to read Jenkinsfile
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/Build/Plugins/GenericEditor@script/a4bd6e68954a25239b55c35ed8919f45766482f2bbd688447c591a1bbf5b1e3e/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-generic-editor.git # timeout=10
Fetching upstream changes from https://github.com/project-sunbird/sunbird-generic-editor.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress -- https://github.com/project-sunbird/sunbird-generic-editor.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse origin/release-4.10.1^{commit} # timeout=10
Checking out Revision 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae (origin/release-4.10.1)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae # timeout=10
Commit message: "Issue #KN-133 merge: Merge pull request #72 from vaibhavbhuva/release-4.10.1"
 > git rev-list --no-walk 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae # timeout=10
[Pipeline] Start of Pipeline
[Pipeline] node
Running on [Jenkins](http://35.223.161.5:8080/computer/(built-in)/) in /var/lib/jenkins/workspace/Build/Plugins/GenericEditor
[Pipeline] {
[Pipeline] ansiColor
[Pipeline] {

[Pipeline] stage
[Pipeline] { (Checkout)
[Pipeline] cleanWs
[WS-CLEANUP] Deleting project workspace...
[WS-CLEANUP] Deferred wipeout is used...
[WS-CLEANUP] done
[Pipeline] checkout
The recommended git tool is: NONE
No credentials specified
Cloning the remote Git repository
Cloning repository https://github.com/project-sunbird/sunbird-generic-editor.git
 > git init /var/lib/jenkins/workspace/Build/Plugins/GenericEditor # timeout=10
Fetching upstream changes from https://github.com/project-sunbird/sunbird-generic-editor.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress -- https://github.com/project-sunbird/sunbird-generic-editor.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-generic-editor.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
Avoid second fetch
 > git rev-parse origin/release-4.10.1^{commit} # timeout=10
Checking out Revision 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae (origin/release-4.10.1)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae # timeout=10
Commit message: "Issue #KN-133 merge: Merge pull request #72 from vaibhavbhuva/release-4.10.1"
[Pipeline] checkout
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-generic-editor.git # timeout=10
Fetching upstream changes from https://github.com/project-sunbird/sunbird-generic-editor.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress -- https://github.com/project-sunbird/sunbird-generic-editor.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse refs/remotes/origin/refs/tags/release-4.10.1_RC1^{commit} # timeout=10
 > git rev-parse refs/tags/release-4.10.1_RC1^{commit} # timeout=10
Checking out Revision 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae (refs/tags/release-4.10.1_RC1)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 7e5d0be37159a68f43365ea4b0faf0f03e05e6ae # timeout=10
Commit message: "Issue #KN-133 merge: Merge pull request #72 from vaibhavbhuva/release-4.10.1"
[Pipeline] sh
+ git rev-parse --short HEAD
[Pipeline] echo
github_release_tag specified, building from github_release_tag: release-4.10.1_RC1
[Pipeline] sh
+ git clone https://github.com/project-sunbird/sunbird-content-plugins.git plugins
Cloning into 'plugins'...
[Pipeline] sh
+ cd plugins
+ git ls-remote --tags origin release-4.10*
+ sort -V
+ grep -o release-4.10.*
+ tail -n1
+ checkout_tag=
+ git checkout tags/ -b
error: switch `b' requires a value
usage: git checkout [<options>] <branch>
   or: git checkout [<options>] [<branch>] -- <file>...

    -q, --quiet           suppress progress reporting
    -b <branch>           create and checkout a new branch
    -B <branch>           create/reset and checkout a branch
    -l                    create reflog for new branch
    --detach              detach HEAD at named commit
    -t, --track           set upstream info for new branch
    --orphan <new-branch>
                          new unparented branch
    -2, --ours            checkout our version for unmerged files
    -3, --theirs          checkout their version for unmerged files
    -f, --force           force checkout (throw away local modifications)
    -m, --merge           perform a 3-way merge with the new branch
    --overwrite-ignore    update ignored files (default)
    --conflict <style>    conflict style (merge or diff3)
    -p, --patch           select hunks interactively
    --ignore-skip-worktree-bits
                          do not limit pathspecs to sparse entries only
    --ignore-other-worktrees
                          do not check if another worktree is holding the given ref
    --recurse-submodules[=<checkout>]
                          control recursive updating of submodules
    --progress            force progress reporting

[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
ERROR: script returned exit code 129
Finished: FAILURE
